### ISSUE TITLE
AWS SSO - net::ERR_CACHE_MISS

### ISSUE THREAD

**Issue by rafaelkonrath** [Noovolari/leapp#65]
**Describe the bug**
After clicked on AWS SSO button on the window "Identity Providers"  a new window is open and is redirect to AWS SSO page but after some seconds the following message below appears and the AWS profiles are not loading.

```[2021-01-19 15:42:54.237] [info] feedURL https://update.electronjs.org/Noovolari/leapp/darwin-x64/0.4.1
[2021-01-19 15:42:54.240] [info] requestHeaders { 'User-Agent': 'Leapp/latest (darwin: x64)' }
[2021-01-19 15:42:54.241] [info] checking-for-update
[2021-01-19 15:42:57.102] [info] update-not-available
[2021-01-19 15:43:22.666] [error] [renderer] [e] net::ERR_CACHE_MISS```

**Leapp Version**
Leapp.
Version 0.4.1 (0.4.1)

**Desktop (please complete the following information):**
 - OS: macOS Catalina
 - OS Version 10.15.5

**Comment by andreacavagna01** [comment]
Thanks for reporting, I will look into it, it's a not handled exception I think

**Comment by andreacavagna01** [comment]
Hi @rafaelkonrath, could I ask you if the portal Url is something like this:
"https://d-123456789uuh0.awsapps.com/start" or you are using an Alias? I think the problem could be the not use of the /start at the end

**Comment by andreacavagna01** [comment]
It's an AWS response gone wrong, I'm going to add the fix in the 0.4.2 release soon with @ericvilla! still have the issue?

**Comment by m-radzikowski** [comment]
Had the same problem. I'm not really sure what's the cause, but after several tries, I finally managed to log in. In the meantime I had logged in to AWS SSO through the browser, but I don't see how this could be related and what action did solve this.

**Comment by andreacavagna01** [comment]
The context of the App and the Browser is different, and it does should not affect how The LogIn flow works with AWS SSO, I will check this issue with @urz9999 to investigate it more deeply.

**Comment by andreacavagna01** [comment]
This issue should be solved in v0.4.3, I will close the issue. @m-radzikowski feel free to open it again if you still have the same problem
